aksharamukha-back/main.py

```diff
@@ -593,9 +591,33 @@ def convert_post():
 
     return text
 
+@app.route('/api/convert_docx', methods=['POST', 'GET'])
+def convert_docx():
+    # TODO remove these lines after adding convert_docx_files method 
+    # to transliterate source code in aksharamukha_python repo
+    
+    # local imports
+    import sys
+    import os
+    # The path to the aksharamukha_python package in system
+    sys.path.append(os.path.abspath(r"..\.."))
+    from aksharamukha_python.aksharamukha.transliterate import convert_docx
+    #################################

```

Comment:
do I understand correctly that this entire section is necessary only until https://github.com/mhdsulaimantan/aksharamukha-python/pull/1 is merged?

Reply:
yes

Reply:
Now, that we've merged https://github.com/mhdsulaimantan/aksharamukha-python/pull/1 

Would it be easier to review/test this PR if it were dependent on the one extending the python module?